src/Raven.Server/Documents/CountersStorage.cs

```diff
@@ -370,9 +368,9 @@ public string IncrementCounter(DocumentsOperationContext context, string documen
                 RemoveTombstoneIfExists(context, documentId, name);
 
                 var etag = _documentsStorage.GenerateNextEtag();
-                var result = ChangeVectorUtils.TryUpdateChangeVector(_documentDatabase.ServerStore.NodeTag, _documentsStorage.Environment.Base64Id, etag, string.Empty);
+                var changeVector = $"{_documentDatabase.ServerStore.NodeTag}:{etag}-{_documentsStorage.Environment.Base64Id}";
```

Comment:
Would be better to move this to `ChangeVectorUtils` in general. Any changes / optimizations would only happen in one place

Reply:
done